website/docs/r/monitor_alert_prometheus_rule_group.html.markdown

```diff
@@ -0,0 +1,189 @@
+---
+subcategory: "Monitor"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_monitor_alert_prometheus_rule_group"
+description: |-
+  Manages an Alert Management Prometheus Rule Group.
+---
+
+# azurerm_monitor_alert_prometheus_rule_group
+
+Manages an Alert Management Prometheus Rule Group.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "West Europe"
+}
+
+resource "azurerm_monitor_action_group" "example" {
+  name                = "example-mag"
+  resource_group_name = azurerm_resource_group.example.name
+  short_name          = "testag"
+}
+
+resource "azurerm_monitor_workspace" "example" {
+  name                = "example-amw"
+  resource_group_name = azurerm_resource_group.example.name
+  location            = azurerm_resource_group.example.location
+}
+
+resource "azurerm_kubernetes_cluster" "example" {
+  name                = "example-cluster"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+  dns_prefix          = "example-aks"
+  kubernetes_version  = "1.25.5"
+
+  default_node_pool {
+    name                   = "default"
+    node_count             = 1
+    vm_size                = "Standard_DS2_v2"
+    enable_host_encryption = true
+  }
+
+  identity {
+    type = "SystemAssigned"
+  }
+}
+
+resource "azurerm_monitor_alert_prometheus_rule_group" "example" {
+  name                = "example-amprg"
+  location            = "West Europe"
+  resource_group_name = azurerm_resource_group.example.name
+  cluster_name        = azurerm_kubernetes_cluster.example.name
+  description         = "This is the description of the following rule group"
+  rule_group_enabled  = false
+  interval            = "PT1M"
+  scopes              = [azurerm_monitor_workspace.example.id]
+  rule {
+    enabled    = false
+    expression = "histogram_quantile(0.99, sum(rate(jobs_duration_seconds_bucket{service=\" billing-processing \"}[5m])) by (job_type))"
```

Comment:
it might be better to use heredoc/multiple line expression here rather then escape "?

Reply:
Thanks for your feedback. I have updated the code. Could you please take another look?